packages/crayons-core/src/components/form/form.tsx

```diff
@@ -640,6 +640,74 @@ export class Form {
     this.values = { ...this.values, [field]: undefined };
   }
 
+  /**
+   * Method to set hidden fields on the form dynamically.
+   *
+   * Note: You must always pass all the fields you wanting to hide. Also, note that the validation for hidden fields will be skipped.
+   *
+   * param: hiddenFields - key value pair of [fieldName]: true | false
+   * example: `setHiddenFields({ first_name: true, last_name: false })`
+   */
+  @Method()
+  async setHiddenFields(hiddenFields: any = {}): Promise<void> {
+    return this._handleFieldModifier('hidden', hiddenFields);
+  }
+
+  /**
+   * Method to set disabled fields on the form dynamically.
+   *
+   * Note: You must always pass all the fields you wanting to disable
+   *
+   * param: disabledFields - key value pair of [fieldName]: true | false
+   * example: `setDisabledFields({ first_name: true, last_name: false })`
+   */
+  @Method()
+  async setDisabledFields(disabledFields: any = {}): Promise<void> {
+    // Transforming disabled to editable
+    const editableFields = Object.keys(disabledFields).reduce(
```

Comment:
You can remove this logic from here and add a check in `_handleFieldModifier` to check if the key is `editable` and negate it.